README.md

```diff
@@ -0,0 +1,112 @@
+
+```
+git remote add origin git@github.com:kokkos/kokkos-core-wiki.git
+```
+
+* **Step 4) Check your remotes; you should see both your fork (`ajpowelsnl`) and the main project repo (`origin`):**<br/>
+
+```
+[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+origin https://github.com/kokkos/kokkos-core-wiki.git (fetch)
+origin https://github.com/kokkos/kokkos-core-wiki.git (push)
+```
+
+* **Step 5) Create a topic branch in your local fork of the kokkos-core-wiki:**<br/>
+
+```
+git checkout -b demo/setup
+```
+
+* **Step 6) Add the Kokkos project Wiki repo from your fork as a remote (you will push changes to this repo to preview before creating a pull request on `origin git@github.com:kokkos/kokkos-core-wiki.git`):**<br/>
+
+```
+git remote add my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git
+```
+
+* **Step 7) Verify your remotes.  You should now have three different repos: your local, forked kokkos-core-wiki (`ajpowelsnl`), the remote kokkos-core-wiki (`origin`), and the Wiki associated with your forked Kokkos project (`my_wiki`):**<br/>
+
+```
+[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git (fetch)
+my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git (push)
+origin https://github.com/kokkos/kokkos-core-wiki.git (fetch)
+origin https://github.com/kokkos/kokkos-core-wiki.git (push)
+```
+
+* **Step 8) Set up your local topic branch to track origin/main:**<br/>
+
+```
+git checkout main
+git branch --set-upstream-to=origin/main 
+git pull origin/main
+git show
+commit f0e10f59f50f23930a2f4eedfce3eed869344277 (HEAD -> main, origin/main)
+Merge: 7b5063e ec9f444
+Author: Damien L-G <dalg24@gmail.com>
+Date:   Mon Feb 28 21:09:11 2022 -0500
+
+Merge pull request #3 from dalg24/fixup_gh_actions
+
+Fixup path gh workflow
+```
+
+* **Step 9) Update your topic branch, rebasing on origin/main:**<br/>
+
+```
+git checkout demo/setup
+git rebase main
+```
+
+* **Step 10) Make the desired changes (on your local topic branch), and push to the main project Wiki (of your fork).  Nota bene: you will need to use the `-f` option to push; this option will overwrite existing files.**<br/>
+
+```
+git push -f my_wiki demo/setup:master
+```
+
+* **Step 11) Preview your changes by navigating to the main project Wiki of your fork:**<br/>
+
+```
+https://github.com/ajpowelsnl/kokkos/wiki
+```
+
+* **Step 12) If your previewed changes are good, push your local topic branch to your fork of the kokkos-core-wiki to create a pull request.  Please note that pushes to your branch will be automatically, immediately deployed to your *fork* of the Kokkos project Wiki.  You cannot push directly to the `main` branch of the remote repo:**<br/>
```

Comment:
I find the second sentence confusing. What are you trying to say? We are not pushing to the Wiki of the fork of `kokkos-core-wiki` here but directly to the fork.

```suggestion
* **Step 12) If your previewed changes are good, push your local topic branch to your fork of the kokkos-core-wiki to create a pull request. You cannot push directly to the `main` branch of the remote repo:**<br/>
```